Config/Media.json

```diff
@@ -1,4 +1,88 @@
-[{
+[
+{
+    "Id":  "2016_x64_Standard_EN_Eval",
+    "Filename":  "2016_x64_EN_Eval.iso",
+    "Description":  "Windows Server 2016 Standard 64bit English",
+    "Architecture":  "x64",
+    "ImageName":  "Windows Server 2016 SERVERSTANDARD",
+    "MediaType":  "ISO",
+    "OperatingSystem":  "Windows",
+    "Uri":  "http://care.dlservice.microsoft.com/dl/download/1/6/F/16FA20E6-4662-482A-920B-1A45CF5AAE3C/14393.0.160715-1616.RS1_RELEASE_SERVER_EVAL_X64FRE_EN-US.ISO",
```

Comment:
Can you change the URL to http://download.microsoft.com/download/1/6/F/16FA20E6-4662-482A-920B-1A45CF5AAE3C/14393.0.160715-1616.RS1_RELEASE_SERVER_EVAL_X64FRE_EN-US.ISO